src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -41,6 +41,7 @@ type
     moduleLoaded: bool
     initialMessagesLoaded: bool
     firstUnseenMessageState: FirstUnseenMessageState
+    scrollToMessageRequests: Table[UUID, string]
```

Comment:
Not sure why do we store the table here instead of just one UUID? Message module shouldn't have more than one request ongoing:
```
  if self.view.getMessageSearchOngoing():
    return
```

Reply:
Indeed, missed that

Reply:
done 👍